src/components/SelectField/SelectField.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react'
+import { Header, Dropdown, DropdownProps } from 'semantic-ui-react'
+import './SelectField.css'
+
+export type SelectFieldProps = DropdownProps & {
+  label?: string
+}
+
+export class SelectField extends React.PureComponent<SelectFieldProps> {
+  hasAction() {
+    let { loading, error, action, onAction } = this.props
```

Comment:
This is a bit of a stretch but most of the code is using `const` for everything except when a variable is re-assigned. I know Field.tsx does the same thing for this method, but maybe we can change both?
Same for `let { label, ...rest } = this.props`